Uplift/Models/SportsDropdown.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  SportsDropdown.swift
+//  Uplift
+//
+//  Created by Artesia Ko on 4/11/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class DropdownData {
+
+    var completed: Bool!
+    var dropStatus: DropdownStatus!
+    var titles: [String]!
```

Comment:
does this need to be a class? we could make it a struct and change all the `var` to `let` in which case you wouldn't need the init